play-ahc-ws-standalone/src/main/java/play/libs/ws/ahc/StandaloneAhcWSRequest.java

```diff
@@ -248,13 +248,20 @@ public StandaloneAhcWSRequest setMethod(String method) {
 
     /**
      * Sets a BodyWritable directly.  See {@link DefaultBodyWritables} for common bodies.
+     * Also sets a Content-Type header if its presented in BodyWritable.
```

Comment:
"Also sets a Content-Type header if its specified in the BodyWritable and is not already set."